### PR TITLE
Fix Nova 4.28.0 removed DangerButton component

### PR DESCRIPTION
**Pull request by dmason30**
As title says the `danger-button` component was removed as Nova switched their buttons to the new `laravel-nova-ui` component library.

This resulted in all the remove buttons rendering like this:
image

### Fix
Changed all of the buttons to use the `laravel-nova-ui` components to match Nova and extract the add/remove buttons to a reusable component.

This PR includes a prod dist build ready for release. 

image

### CODE REVIEW

**Comment by michaelklopf** [comment]
We have the same issue. I pulled the PR branch and tested it with our implementation and it works

image

File: resources/js/components/ExceptionsTable.vue [hunk 1]
```diff
@@ -39,24 +39,25 @@
                     <div v-else>{{ __('Closed') }}</div>
                 </table-column>
                 <table-column v-if="editable" class="text-right">
-                    <default-button @click.prevent="$emit('addInterval', 'exceptions', exception.date)"><span class="px-1">+</span></default-button>
-                    &nbsp;
-                    <danger-button @click.prevent="$emit('removeException', exception.date)"><span class="px-1">-</span></danger-button>
+                    <add-button @click.prevent="$emit('addInterval', 'exceptions', exception.date)" />                    &nbsp;
```
Comment (michaelklopf) [inline resources/js/components/ExceptionsTable.vue]: The `&nbsp;` is floating on the right there, can it be brought into the next line?

**Comment by michaelklopf** [comment]
@dmason30 

Hey, I wanted to use your branch for now and added it to the composer.json, but the delete button didn't appear. I tested the commit `daa004b`, and with this one it still works. So something must have failed during the build step.

**Comment by dmason30** [comment]
> @dmason30
> 
> Hey, I wanted to use your branch for now and added it to the composer.json, but the delete button didn't appear. I tested the commit `daa004b`, and with this one it still works. So something must have failed during the build step.

Rendering for me without issue.

Although seems an update in 4.32.x has is now making the time inputs full width, I would rather this got merged and fix that as a separate PR. 

image

**Comment by timdiels1** [comment]
I can confirm this is working for Laravel Nova 4.32.12
Indeed the styling is broken again, but as suggested that can fixed in a new PR.

**Comment by michaelklopf** [comment]
Thank you, now it works, I still don't know what happened over here.

**Pull request merged**